src/MetaMorpho.sol

```diff
@@ -823,16 +823,14 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     }
 
     /// @dev Accrues the fee and mints the fee shares to the fee recipient.
-    /// @return newTotalAssets The new vault's total assets.
-    function _accrueFee() internal returns (uint256 newTotalAssets) {
+    /// @return accruedTotalAssets The vaults total assets after accruing the interest.
```

Comment:
I actually find the naming `accruedTotalAssets` to be confusing, because I would expect it to correspond to `totalAssets() - lastTotalAssets` and not only `totalAssets()` as it is now

Am I the only one?

Reply:
I see what you mean but with `total` in the name it should lead in the right direction no ? 

Do you have a suggestion ?

Reply:
`Total` doesn't lead me to the right direction still, because I would think that it corresponds to the "accrued difference of total assets"

But I'm the only one then I guess I'm too biased

I preferred the naming "newTotalAssets" which I agree can be confusing from a macro perspective where you have 2 "new total assets" values for each interaction but in the context of the event it may be fine because you only have a single possible "new total assets" after having accrued interest

Reply:
Having twice the same name is confusing to me. What do you think about `totalAssetsAfterAccrual` ?

Reply:
Or `afterAccrualTotalAssets`, to stay consistent with `lastTotalAssets`and `newTotalAssets` that prefix `TotalAssets`

Reply:
It works, though it's verbose

Reply:
`updatedTotalAssets`?

Reply:
> Where is the name used twice?

In the event `UpdateLastTotalAssets`.

> `updatedTotalAssets`?

This is also confusing, because it is not the value that is used to update total assets stored (and there is the `UpdateLastTotalAssets` event that we could think is linked to it)

Reply:
So instead I just changed the event `UpdateLastTotalAssets` to use the name `updatedTotalAssets`, which sounds obvious now :)